guides/source/active_record_basics.md

```diff
@@ -377,9 +475,7 @@ Callbacks
 ---------
 
 Active Record callbacks allow you to attach code to certain events in the
-life-cycle of your models. This enables you to add behavior to your models by
-transparently executing code when those events occur, like when you create a new
-record, update it, destroy it, and so on.
+life-cycle of your models. This enables you to add behavior to your models by executing code when those events occur, like when you create a new record, update it, destroy it, and so on.
```

Comment:
Nit: In Active Model callbacks we used `lifecycle`, and other places are `life cycle`. Maybe we stick to one (at least in relation to callbacks) for consistency? (cc @Ridhwana)

Reply:
I'll go with "lifecycle"